cypress/integration/Tabs.spec.ts

```diff
@@ -4,13 +4,14 @@ describe('Tabs', () => {
   before(() => {
     h.stories.visit();
   });
-  ['Simple', 'NamedKeys'].forEach(story => {
+
+  ['Simple', 'Named Keys'].forEach(story => {
     context(`given the '${story}' story is rendered`, () => {
       beforeEach(() => {
         h.stories.load('Testing/React/Labs/Tabs', story);
       });
 
-      it('should pass accessibility checks', () => {
+      it('should pass axe checks', () => {
         cy.checkA11y();
```

Comment:
I might change the name here to `cy.checkAxe`. Axe does not check all accessibility issues, mostly low-hanging fruit or obvious issues.

Reply:
I agree with this idea.